tools/otci/otci/otci.py

```diff
@@ -2619,6 +2638,165 @@ def wait_for(self, command: str, expect_line: Optional[Union[str, Pattern, Colle
     # TODO: pskc [-p] <key>|<passphrase>
     #
 
+    #
+    # Platform Commands Utilities
+    #
+    def support_iperf3(self) -> bool:
+        """Check whether the platform supports iperf3."""
+        #
+        # Iperf3 help example:
+        #
+        # $ iperf3 --help
+        # Usage: iperf3 [-s|-c host] [options]
+        #       iperf3 [-h|--help] [-v|--version]
+        #
+        #       Server or Client:
+        #         -p, --port      #         server port to listen on/connect to
+        #       ...
+        #
+        ret = False
+        output = self.execute_platform_command('iperf3 --help')
+        if len(output) > 0 and 'Usage: iperf3 [-s|-c host] [options]' in output[0]:
```

Comment:
It may not be a good idea to check the help output of `iperf3` since it may change among variants. 

What about simply check the existence of this binary? E.g. check the return value of `command -v iperf3`.